test/e2e/common/configmap_volume.go

```diff
@@ -540,6 +540,18 @@ var _ = Describe("[sig-storage] ConfigMap", func() {
 		})
 
 	})
+
+	//Slow (~5 mins)
+	It("non-optional updates should not reflected in volume [Slow]", func() {
+		volumeMountPath := "/etc/configmap-volumes"
+		podName := "pod-configmaps-" + string(uuid.NewUUID())
+		Expect(f.WaitForPodRunning(createNonOptionalConfigMapPod(f, volumeMountPath, podName).Name)).To(HaveOccurred(), "created pod %q with non-optional configMap in namespace %q", podName, f.Namespace.Name)
```

Comment:
this seems like a lot of nesting, I would pull the `f.WaitForPodRunning` out above the expect and `Expect(err)`